src/main/java/Duke.java

```diff
@@ -1,10 +1,224 @@
+import duke.*;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+/**
+ * Duke Program behaves like a chatbot that interacts with user input.
+ * It allows the user to create a list of Tasks consisting of todos, deadlines, and events
+ * @author oliverloo
+ * @version 1.0
+ *
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    /**
+     * Main method to initiliase duke chatbot
+     * @param args
+     */
+        public static void main (String[]args) {
+            Tasklist list = new Tasklist();
+            try {
+                Storage.loadFile(list);
+            } catch(Exception e) {
```

Comment:
A catch for an "everything" Exception, maybe you could instead make Storage.loadFile throw a more specific DukeException instead?